hphp/ppc64-asm/decoder-ppc64.h

```diff
@@ -461,13 +461,7 @@ class Decoder : private boost::noncopyable {
     }
   }
 
-  /*
-   * Disable optimizations for this constructor.  In release mode -O3 causes
-   * compilation to hang due the huge initialization list.  This is a static
-   * singleton constructor, it's only called once and, when trace is enabled so
-   * optimization here is not a big issue.
-   */
-  NO_OPT Decoder();
+  Decoder();
```

Comment:
@Orvid  there' s no `NO_OPT` on the implementation (.cpp): https://github.com/facebook/hhvm/blob/master/hphp/ppc64-asm/decoder-ppc64.cpp#L155